src/main/java/io/jenkins/plugins/luxair/ImageTagParameterConfiguration.java

```diff
@@ -46,6 +47,8 @@ public Ordering getDefaultTagOrdering() {
         return defaultTagOrdering != null ? defaultTagOrdering : Ordering.NATURAL;
     }
 
+    public Boolean getDefaultVerifySsl() { return defaultVerifySsl; }
```

Comment:
```suggestion
    public Boolean getDefaultVerifySsl() { return defaultVerifySsl == null || defaultVerifySsl; }
```
If the global configuration does not get edited after this update then `defaultVerifySsl` will return as `null`.
(+this unifies the getter with the others :wink: )